src/bot.py

```diff
@@ -123,6 +127,36 @@ async def calc_command(ctx: discord.ApplicationContext):
     await ctx.send_modal(slash_commands.calc.Modal(title="Enter Checkin Details"))
 
 
+@bot.slash_command(name="bentest", description="Test command for Ben")
+async def bentest_command(ctx: discord.ApplicationContext):
+    await ctx.respond("Test successful")
+
```

Comment:
```suggestion
```

We don't actually need this. If we want something like it maybe add a ping command and have it print so you can see in log which instance it hits?